File: examples/java/kora-java-resilient/src/test/java/ru/tinkoff/kora/example/resilient/ComboServiceTests.java
package ru.tinkoff.kora.example.resilient;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;
import ru.tinkoff.kora.test.extension.junit5.KoraAppTest;
import ru.tinkoff.kora.test.extension.junit5.TestComponent;

@KoraAppTest(Application.class)
class ComboServiceTests {

    @TestComponent
    private ComboService comboService;

    @Test
    void test() {
        assertNotNull(comboService.getValue(false));
    }
}

File: examples/kotlin/kora-kotlin-resilient/src/test/kotlin/ru/tinkoff/kora/kotlin/example/resilient/ComboServiceTests.kt
package ru.tinkoff.kora.kotlin.example.resilient

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test
import ru.tinkoff.kora.test.extension.junit5.KoraAppTest
import ru.tinkoff.kora.test.extension.junit5.TestComponent

@KoraAppTest(Application::class)
class ComboServiceTests {

    @TestComponent
    lateinit var comboService: ComboService

    @Test
    fun test() {
        assertNotNull(comboService.getValue(false))
    }
}
